docs/streams.html

```diff
@@ -894,6 +894,19 @@
            If you want to view the changelog stream of the  <code>KTable</code> then you could call <code>KTable.toStream().print()</code>.
         </p>
 
+        <p> Metrics using exactly-once semantics: </p>
+        <p>
+            If exactly-once processing is enabled via parameter <code>processing.guarantees</code>, internally Streams switches from a producer per thread to a producer per task runtime model.
```

Comment:
"If exactly-once processing is enabled via the <code>processing.gaurantees</code> parameter, ..."